.github/workflows/ci.yml

```diff
@@ -79,7 +79,8 @@ jobs:
                   smoke/test_metrics_api
                   smoke/test_migration
                   smoke/test_multipleips_per_nic
-                  smoke/test_nested_virtualization",
+                  smoke/test_nested_virtualization
+                  smoke/test_set_source_nat",
```

Comment:
test_set_source_nat.py smoke test file is missing !

Reply:
oops